manifests/istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -95,6 +101,7 @@ data:
 
     {{- if .Values.telemetry.v2.enabled }}
     disableMixerHttpReports: true
+    mixerReportServer: ""
```

Comment:
Is this trying to redefine this key ? It may work with parsing but would be confusing

Reply:
No, just setting it empty